### PR TITLE
Add build process for client development

### PR DESCRIPTION
**Pull request by chrispicato**
Add build process for client development

### CODE REVIEW

File: compiled/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!doctype html>
```
Comment (andrewk17) [inline compiled/index.html]: should we ignore the compiled folder and not merge this in?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html>
+
```
Comment (andrewk17) [inline index.html]: any reason for this spacing across the document?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    "reactjs",
+    "boilerplate",
+    "hot",
+    "reload",
+    "hmr",
+    "live",
+    "edit",
+    "webpack"
+  ],
+  "author": "Dan Abramov <dan.abramov@me.com> (http://github.com/gaearon)",
```
Comment (andrewk17) [inline package.json]: change authors

**Review by kevinbrosamle — APPROVED** [review]

**Review by andrewk17 — CHANGES_REQUESTED** [review]

**Review by tetratorus — APPROVED** [review]
BOOM

**Review by andrewk17 — APPROVED** [review]

**Pull request merged**